single-sign-on/index.md

```diff
@@ -23,46 +18,78 @@ To enable SSO in Docker Hub, you need the following:
 In addition, all email addresses should be added to your IdP.
 
-## Configure SSO
+## Configure
 
 To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} to obtain the **ACS URL** and **Entity IDs** to complete the IdP server configuration process. You can only configure SSO with a single IdP.  When this is complete, log back into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and complete the SSO enablement process.
 
-### Identity provider configuration
+### SAML 2.0 IdP configuration
+
+1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to **Organizations** and select the organization that you want to enable SSO on.
+2. Click **Settings** and select the **Security** tab.
+3. Select an authentication method for **SAML 2.0**.
+
+    ![SSO SAML1](images/sso-saml1.png){:width="500px"}
+
+4. In the Identity Provider Set Up, copy the **Entity ID**, **ACS URL** and **Certificate Download URL**.
+
+    ![SSO SAML2](images/sso-saml2.png){:width="500px"}
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to Organizations and select the organization that you want to enable SSO on.
-2. Click **Settings** and select the Security tab.
-3. Select an authentication method based on your identity provider. Docker currently supports **SAML 2.0** and **Azure AD**.
-4. Copy the ID and/or URL in the **Identity Provider Set Up**.
-    For SAML 2.0, copy the **Entity ID** and **ACS URL**. For Azure AD, copy your **Redirect URL/Reply URL**.
 5. Log into your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+> **Note:**
+> the NameID is your email address and is set as the default.
+> For example, <Subject><NameID>yourname@mycompany.com</NameID>.
+
 6. Complete the fields in the **Configuration Settings** section and click **Save**. If you want to change your IdP, you must delete your existing provider and configure SSO with your new IdP.
 
-![SSO SAML](images/sso-saml.png){:width="500px"}
+    ![SSO SAML3](images/sso-saml3.png){:width="500px"}
+
+### Azure AD IdP configuration
+
+1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to **Organizations** and select the organization that you want to enable SSO on.
+2. Click **Settings** and select the **Security** tab.
+3. Select an authentication method for **Azure AD**.
+
+    ![SSO Azure1](images/sso-azure1.png){:width="500px"}
+
+4. In the Identity Provider Set Up, copy the **Redirect URL / Reply URL**.
+
+    ![SSO Azure2](images/sso-azure2.png){:width="500px"}
+
+5. Log into your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+> **Note:**
```

Comment:
The Note here resets the numbered list. Indenting the note should fix this issue